frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+        <button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-none d-md-inline" data-toggle="collapse" data-target="#searchbar-collapse" aria-expanded="false" aria-label="Toggle Search">
+          <span class="fa fa-search" aria-hidden="true"></span>
+        </button>
+        <!--Translation Button-->
+        <div class="dropdown">
+          <button class="dropbtn">Lenguaje</button>
+          <div class="dropdown-content">
+            <a href="#" onclick="getEnglishPage()">English</a>
+            <a href="#" onclick="">Español</a>
+            <script >
+              function getEnglishPage(){
+                window.location.href = "index.html";
+              }
+            </script>
+          </div>
+        </div>
+      </div>
+
+    </nav>
+    </div><!--container-->
+    </div><!--program-header-->
+    <!-- mobile search button -->
+    <button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-md-none" data-toggle="collapse" data-target="#searchbar-collapse-mobile" aria-expanded="false" aria-label="Toggle Search">
+    <span class="fa fa-search" aria-hidden="true"></span>
+    </button>   
+    <!-- Start collapsible site search bar -->
+    <div class="site-search-bar">
+    <div class="container">
+    <div class="row">
+      <div class="col">
+        <nav class="navbar navbar-light">
+          <div class="navbar-collapse collapse" id="searchbar-collapse">
+            <form
+                method="get"
+                class="site-searchbar-form form-inline ThunderstoneForm"
+                action="https://search4.nrel.gov/texis/search/" 
+                id="ThunderstoneSearchForm">
+
+              <input type="hidden" name="pr" value="metanrel" />
+              <div class="form-group">
+                  <label for="query" class="sr-only">Buscar NREL.gov</label>
+                  <input
+                      type="search"
+                      id="query" name="query"
+                      maxlength="2047"
+                      placeholder="Buscar NREL.gov"
+                      class="form-control"
+                      aria-label="Buscar NREL.gov">
+                  <button type="submit" class="btn btn-default">Buscar</button>
+              </div>
+            </form>
+          </div>
+        </nav>
+      </div>
+    </div>
+    </div>
+    </div>    
+    <!-- End collapsible site search bar -->
+    <!--startindex-->
+
+		<div class="container" id="content" role="main">
+         <div class="row">
+            <div class="col-lg-9">
+               <h1>Universidad de Puerto Rico Recinto de Mayagüez Proyecto NSF CIVIC</h1>
+               <!--<h1 id="studyTitleOne"></h1>--> <!--This was removed do to lack of translated token at .json file. This is something that we will have to discuss with Shankari.-->
+               <p class="lead">Gracias por participar en la Universidad de Puerto Rico Recinto de Mayagüez Proyecto NSF CIVIC</p>
```

Comment:
Ditto

Reply:
Same thing applies here. It will say "Gracias por participar en la University of Puerto Rico at Mayagüez Proyecto UPRM CIVIC"

Reply:
Same here; this also uses the deployment partner name, so if you add a spanish version of the deployment partner name, you should be able to fix this as well.